github-actions/rebase-pr/entrypoint.sh

```diff
@@ -8,11 +8,14 @@ git config --global user.name "Angular Rebase Bot"
 # Exit code for a successful run
 SUCCESS_EXIT_CODE=0
 
-# The repository name and branch name for the PRs head and base
+# Information taken from github event object
 PR_REPOSITORY=$(jq -r ".pull_request.head.repo.full_name" "$GITHUB_EVENT_PATH")
 PR_BRANCH=$(jq -r ".pull_request.head.ref" "$GITHUB_EVENT_PATH")
 TARGET_REPOSITORY=$(jq -r ".pull_request.base.repo.full_name" "$GITHUB_EVENT_PATH")
 TARGET_BRANCH=$(jq -r ".pull_request.base.ref" "$GITHUB_EVENT_PATH")
+REPO_FULL_NAME=$(jq -r ".repository.full_name" "$GITHUB_EVENT_PATH")
+PR_NUMBER=$(jq -r ".pull_request.number" "$GITHUB_EVENT_PATH")
+REBASE_PR_LABEL=$(jq -r ".label.name" "$GITHUB_EVENT_PATH")
```

Comment:
```suggestion
REBASE_PR_LABEL=$(jq -r ".label.name | @uri" "$GITHUB_EVENT_PATH")
```

This should provide the url encoded version of the string